api/client/service/update.go

```diff
@@ -38,12 +38,12 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.String("args", "", "Service command args")
 	addServiceFlags(cmd, opts)
 
-	flags.Var(newListOptsVar(), flagEnvRemove, "Remove an environment variable")
-	flags.Var(newListOptsVar(), flagLabelRemove, "Remove a label by its key")
-	flags.Var(newListOptsVar(), flagContainerLabelRemove, "Remove a container label by its key")
-	flags.Var(newListOptsVar(), flagMountRemove, "Remove a mount by its target path")
-	flags.Var(newListOptsVar(), flagPublishRemove, "Remove a published port by its target port")
-	flags.Var(newListOptsVar(), flagConstraintRemove, "Remove a constraint")
+	flags.Var(newListOptsVar(nil), flagEnvRemove, "Remove an environment variable")
```

Comment:
Thanks @cpuguy83. The pull request has been updated. Please let me know if there are any issues.